Gems/Atom/Feature/Common/Assets/Passes/MainPipeline.pass

```diff
@@ -41,7 +41,10 @@
                 },
                 {
                     "Name": "RayTracingAccelerationStructurePass",
-                    "TemplateName": "RayTracingAccelerationStructurePassTemplate"
+                    "TemplateName": "RayTracingAccelerationStructurePassTemplate",
+                    "ExecuteAfter": [
+                        "SkinningPass"
```

Comment:
This does not guarantee execution order on the gpu. In fact we should remove support for ExecuteAfter/ExecuteBefore completely as it gives a false sense of dependency to the developer. The gpu execution dependency can only be established via framegraph, so in this case this dependency should only work if the output of skinning pass is an input attachment for the RT pass.

Reply:
It depends on what you are trying to do.
- Option 1 - You want to use gpu skinned data in the same frame to be used for RT later in the frame. For this you will need to run the skinning pass and then either build the BLAS within the gpu or ferry the data over to the cpu to build it on cpu (which can get very costly) and then use the data on the gpu. This will be the most complicated and expensive as you will have to bring data from gpu over to the cpu setup correct dependencies for the skinning and RT passes appropriately.

- Option 2 - You do skinning on cpu and then build the BLAS on the cpu which is a reasonable thing but if you are cpu bound this can get tricky if you want RT on skinned meshes. This may be easier to setup but would be a lot of work to add cpu side skinning.

- Option 3 - You use gpu skinned data from previous frame/s. This means that the data will not be accurate but it could work given that we are talking about indirect lighting. For this option you build out BLAS structures from skinned mesh on the cpu at the **end** of the frame. Then next frame you should be able to use the BLASes as you wish and would hopefully not require setting up any dependencies with the skinning pass as it can run at any time independent of RT pass. Tagging @dmcdiarmid-ly in case I missed something.

Reply:
Will resolve this thread now that this has been addressed with the latest changes.